modules/osdk-cli-reference-olm-uninstall.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-olm-uninstall_{context}"]
+= uninstall
+`olm uninstall` subcommand uninstalls OLM from your cluster.
+
+
+.`uninstall` arguments
+[options="header",cols="1,3"]
+|===
+|Argument |Description
+
+|`--olm-namespace` string
```

Comment:
For consistency, should the two instances of "string" in this table be enclosed in parens?

Reply:
nice catch. Missed it!